src/atomate2/vasp/flows/phonons.py

```diff
@@ -0,0 +1,318 @@
+            # angels
+            conv_job = structure_to_conventional(structure, self.symprec)
+            jobs.append(conv_job)
+            structure = conv_job.output
+
+        # if supercell_matrix is None, supercell size will be determined
+        if supercell_matrix is None:
+            supercell_job = get_supercell_size(
+                structure,
+                self.min_length,
+                self.prefer_90_degrees,
+                **self.get_supercell_size_kwargs,
+            )
+            jobs.append(supercell_job)
+            supercell_matrix = supercell_job.output
+
+        if self.bulk_relax_maker is not None:
+            # optionally relax the structure
+            bulk = self.bulk_relax_maker.make(structure, prev_vasp_dir=prev_vasp_dir)
+            jobs.append(bulk)
+            structure = bulk.output.structure
+
+        # get a phonon object from phonopy
+        displacements = generate_phonon_displacements(
+            structure=structure,
+            supercell_matrix=supercell_matrix,
+            displacement=self.displacement,
+            sym_reduce=self.sym_reduce,
+            symprec=self.symprec,
+            use_symmetrized_structure=self.use_symmetrized_structure,
+            kpath_scheme=self.kpath_scheme,
+            code=self.code,
+        )
+        jobs.append(displacements)
+
+        # perform the phonon displacement calculations
+        vasp_displacement_calcs = run_phonon_displacements(
+            displacements=displacements.output,
+            structure=structure,
+            supercell_matrix=supercell_matrix,
+            phonon_maker=self.phonon_displacement_maker,
+        )
+        jobs.append(vasp_displacement_calcs)
+
+        # Computation of static energy
+        if self.static_energy_maker is not None and total_dft_energy is None:
+            static_job = self.static_energy_maker.make(structure=structure)
+            jobs.append(static_job)
+            total_energy = static_job.output.output.energy
+            static_run_job_dir = static_job.output.dir_name
+            static_run_uuid = static_job.output.uuid
+        else:
+            total_energy = total_dft_energy
+            static_run_job_dir = None
+            static_run_uuid = None
+
+        # Computation of BORN charges
+        if self.born_maker is not None and (born is None or epsilon_static is None):
+            born_job = self.born_maker.make(structure)
+            jobs.append(born_job)
+
+            # I am not happy how we currently access "born" charges
+            # This is very vasp specific code
+            epsilon_static = born_job.output.calcs_reversed[0].output.epsilon_static
+            born = born_job.output.calcs_reversed[0].output.outcar["born"]
+            born_run_job_dir = born_job.output.dir_name
+            born_run_uuid = born_job.output.uuid
+        else:
+            epsilon_static = epsilon_static
+            born = born
```

Comment:
Remove these lines

Reply:
In Germany, we call this "betriebsblind". Thanks.

Reply:
That's a great word!